.eslintrc.json

```diff
@@ -15,12 +15,25 @@
   "extends": [
     "eslint:recommended",
     "plugin:prettier/recommended",
-    "plugin:@typescript-eslint/eslint-recommended"
+    "plugin:@typescript-eslint/recommended"
   ],
   "rules": {
     "no-unused-vars": "off",
     "@typescript-eslint/no-unused-vars": ["error", {
       "argsIgnorePattern": "^_"
-    }]
-  }
+    }],
+    "@typescript-eslint/ban-types": "off",
+    "@typescript-eslint/no-explicit-any": "off",
+    "@typescript-eslint/no-empty-function": "off",
+    "@typescript-eslint/no-empty-interface": "off",
```

Comment:
2 problems. 

Turning an interface into a type alias will require turning it back as soon as a new property comes up that is unnecessary work IMO.

```ts
interface ElementMixin extends DirMixin {}
```
vs
```ts
type ElementMixin = DirMixin
```

Reply:
Yeah I don't see the benefit here, on the other hand using interfaces everywhere is consistent.